packages/iTwinUI-react/src/core/ThemeProvider/ThemeProvider.test.tsx

```diff
@@ -293,4 +293,16 @@ describe('Fallback (without children)', () => {
     expect(document.documentElement.dataset.iuiContrast).toBeUndefined();
     expect(document.body.classList).not.toContain('iui-root');
   });
+
+  it('should not modify root if <body> has theme', () => {
+    render(
+      <body data-iui-theme='dark'>
+        <ThemeProvider theme='light' />
+      </body>,
+      { container: document.documentElement },
```

Comment:
A little confused why is this needed? What does this line do?

Reply:
normally, testing-library will add the element inside a div and use that as a container. this line is telling it to use `<html>` as the container since we want `<body>` to be a direct child of it.

when confused, you can always look up the api docs: https://testing-library.com/docs/react-testing-library/api/#container

Reply:
> ... and this is wrong since there are two `<body>` and also since `<body>` is a child of a `<div>`?

Yeah, we only want one `<body>` and it should be in the correct place.
 
> So with this line, is the test dom exactly what we provide? So something like this?

It will be like this (includes the container, and ThemeProvider doesn't render anything):
```html
<html>
  <body data-iui-theme='dark'>
  </body>
</html>
```

> Also just curious, is there a way to `console.log` the dom tree in a easily readable form and not like a JS object type?

You can use `debug` which is returned by `render`.
```js
    const { debug } = render(...);
    debug();
```